src/pages/admin/event/[eventId]/[year]/edit.tsx

```diff
@@ -0,0 +1,189 @@
+import React, { useEffect, useState } from 'react';
+import { NextPage } from 'next';
+import { useRouter } from 'next/router';
+import { EventForm } from '@/components/Events/EventForm';
+import { EventFormSchema } from '@/components/Events/EventFormSchema';
+import { useToast } from "@/components/ui/use-toast";
+import { fetchBackend } from '@/lib/db';
+import { v4 as uuidv4 } from 'uuid';
+
+const EditEventPage: NextPage = () => {
+    const { toast } = useToast();
+    const router = useRouter();
+    const { eventId, year } = router.query;
+    const [initialData, setInitialData] = useState<Partial<EventFormSchema>>();
+    const [isLoading, setIsLoading] = useState(true);
+
+    // Transform backend question format to frontend format
+    const transformBackendQuestion = (q: any) => ({
+        id: uuidv4(), // Generate new ID for form management
+        type: q.type,
+        question: q.label, // Convert back to v2 format
+        required: q.required,
+        options: q.choices ? q.choices.split(',') : [], // Convert string back to array
+        charLimit: q.charLimit,
+        questionImageUrl: q.questionImageUrl || '',
+        participantCap: q.participantCap,
+        isSkillsQuestion: q.isSkillsQuestion
+    });
+
+    useEffect(() => {
+        const fetchEventData = async () => {
+            if (!eventId || !year) return;
+            
+            setIsLoading(true);
+            try {
+                const data = await fetchBackend({
+                    endpoint: `/events/${eventId}/${year}`,
+                    method: 'GET'
+                });
+                
+                console.log('Received data:', data);
```

Comment:
do we wanna keep these logs?

Reply:
@AllanT102 Would to prefer to keep these client-side logs for now and remove them after acceptance testing, before launch